components/layout/setup-overview.tsx

```diff
@@ -72,8 +72,8 @@ const OverviewTemplate = (props) => {
             <h1>Getting Started With Tina</h1>
             <hr />
             <blockquote>
-              TinaCMS supports the latest LTS version of Node. We currently
-              support v18, v20, and v22 of Node with NPM, PNPM, and Yarn.
+              TinaCMS requires Node version 18 or higher, including all LTS
+              releases, and supports NPM, PNPM, and Yarn.
```

Comment:
```suggestion
              TinaCMS is supported only on currently maintained LTS versions of Node.js and works with NPM, PNPM, and Yarn.
```

Reply:
Is it worth stating the minimum version is 18 or do you think this covers it

Reply:
18 will not be supported after April so I'd avoid adding in the maintenance debt